compiler/stable_mir/src/mir/pretty.rs

```diff
@@ -24,10 +24,12 @@ impl Debug for Place {
 
 pub(crate) fn function_body<W: Write>(writer: &mut W, body: &Body, name: &str) -> io::Result<()> {
     write!(writer, "fn {name}(")?;
-    body.arg_locals()
-        .iter()
-        .enumerate()
-        .try_for_each(|(index, local)| write!(writer, "_{}: {}", index + 1, local.ty))?;
+    let mut sep = "";
+    body.arg_locals().iter().enumerate().try_for_each(|(index, local)| {
```

Comment:
frankly i dont know why this isn't just written with a `for` loop -- currently this looks significantly worse.

Reply:
Do you want me to change it here?

Reply:
Nah, it's not worth the churn